bin/upgrade_version

```diff
@@ -17,7 +17,7 @@
 UPVERSION_LIST=etc/upversion.txt
 TESTS_REGEX="^(\s{0,4}\"version\"\s*:\s*)([0-9.\"]*)(,?)"
 SCHEMA_VERSION_IDENTIFIER=\$udmi_version # must start with $
-ERROR_MARKER=.upversion_error
```

Comment:
does this change actually do anything?   I have no problem with it -- more just a curiosity

Reply:
Nope, was testing to see if it did and just left it